README.md

```diff
@@ -41,4 +41,4 @@ Maintainers ([@open-telemetry/specs-semconv-maintainers](https://github.com/orgs
 
 *Find more about the maintainer role in [community repository](https://github.com/open-telemetry/community/blob/master/community-membership.md#maintainer).*
 
-[SpecificationVersion]: https://github.com/open-telemetry/opentelemetry-specification/tree/v1.21.0
+[SpecificationVersion]: https://github.com/open-telemetry/opentelemetry-specification/tree/v1.22.0
```

Comment:
Should we wait for 1.23.0 (https://github.com/open-telemetry/opentelemetry-specification/pull/3584) or go ahead with 1.22.0?

Reply:
It's ~2min to bump versions with the script after checking the changelog.  I'm ok doing this a second time if needed.